core/domain/exp_domain.py

```diff
@@ -81,9 +81,20 @@
 CMD_RENAME_STATE = 'rename_state'
 # This takes an additional 'state_name' parameter.
 CMD_DELETE_STATE = 'delete_state'
-# This takes additional 'state_name', 'content_id', 'language_code' and
-# 'content_html' and 'translation_html' parameters.
+# TODO(#12981): Write a one-off job to modify all existing translation
+# suggestions to use CMD_ADD_WRITTEN_TRANSLATION.
+# DEPRECATED: This command is deprecated. Please do not use. The command remains
+# here to support old suggestions. This takes additional 'state_name',
+# 'content_id', 'language_code' and 'content_html' and 'translation_html'
+# parameters.
 CMD_ADD_TRANSLATION = 'add_translation'
+# This takes additional 'state_name', 'content_id', 'language_code',
+# 'needs_update', 'date_format', 'content_html' and 'translation_html'
```

Comment:
```suggestion
# 'needs_update', 'date_format', 'content' and 'translation'
```

As now we have data_format in the command which will help decide the format of the content/translation.

Reply:
Update: I am changing it back to the original ExplorationChange dict (except for the change in commit command) because changing content_html and translation_html will result all the existing ExplorationChange objects becoming invalid. Changing all the places content_html and translation_html is used is out of scope of this PR because we rely on translation_html and content_html in multiple flows in the code base -- it will involve adding too many special casing (to handle existing data change objects -- for the accept/reject scenario, updating while reviewing, and all the places the suggestion is validated). It is better to do this as part of the TODO task where existing suggestion will be migrated.